catalog/app/utils/GlobalAPI.tsx

```diff
@@ -0,0 +1,68 @@
+import invariant from 'invariant'
+import * as React from 'react'
+import * as rredux from 'react-redux'
+import type * as redux from 'redux'
+
+import * as Auth from 'containers/Auth'
+import defer from 'utils/defer'
+
+export function GlobalAPIProvider({
```

Comment:
I realized, that I need to rename JsonEditor to JSONEditor

Reply:
well, that's not necessary, since we're not 100% settled on acronym capitalization rules -- both approaches are equally valid (depending on the context)